app/views/dashboard.pug

```diff
@@ -48,39 +48,31 @@ html(lang="en")
-                        div#contactsHeading(style='position:absolute;width:100%;height:10%;left:0%;top:0%;')
-                            div(name='contactsPopUpTitle' style='position:absolute;width:95%;height:50%;left:0%;top:25%;')
-                                span My Contacts
-                                button.btn.btn-default#closeContactsButton(style='position:absolute;width:height;height:100%;right:0%;top:0%;') X
-
-                        div(name='tableDiv' style='position:absolute;width:100%;height:80%;left:0%;top:10%;')
+            // Contacts Modal
+            div.modal#contactsModal(role='dialog')
+                div.modal-dialog.modal-lg
+                    div.modal-content
+                        div.modal-header
+                            button.close(type='button' data-dismiss="modal") &times;
+                            h4.modal-title My Contacts
+                        div.modal-body
                             table.table#contactsTable
-
-                // Compose Message Tab
-                div.tab.pop-up#composeTab
-                    div.pop-up-contents#composeView
-
-                        div#composeTopBar
-                            span#composeHeading New Message
-                            button.btn.btn-default#closeComposeButton X
-
-                        div#composeTitle(style='position:absolute;width:100%;height:100%;left:0%;top:5%;')
-                            span Title
-
-                        div(name='composeHeading' style='position:absolute;width:100%;height:5%;left:0%;top:10%;')
-                            textarea(style='position:absolute;width:96%;height:100%;left:2%;top:0%;background-color:#f5e9e5;')
-
-                        div#composeContacts(style='position:absolute;width:100%;height:5%;left:0%;top:15%;')
-                            div(name='composeContactsButton' style='position:absolute;width:20%;height:100%;left:80%;top:0%;')
-                                button.btn.btn-default.contacts(type='submit' style='position:absolute;width:100%;height:100%;left:0%;top:0%;') Select Recipient(s)
-
-                        div#composeMessage(style='position:absolute;width:100%;height:75%;left:0%;top:20%;')
-                            div(name='composeHeading' style='position:absolute;width:100%;height:5%;left:0%;top:0%;')
-                                span Message
-                            textarea(style='position:absolute;width:96%;height:95%;left:2%;top:5%;background-color:#f5e9e5;')
-
-                        div#composeSend(style='position:absolute;width:100%;height:5%;left:0%;top:95%;')
-                            button.btn.btn-default#send(type='submit' style='position:absolute;width:20%;height:100%;left:80%;top:0%;') Send
+                        div.modal-footer
+                            button.btn.btn-default(type='button' data-dismiss='modal' data-toggle='modal' data-target='#composeModal') Choose Recipient(s)

```

Comment:
Now that I think about it, this button text doesn't make much sense, maybe change it to compose? or something?